libflux/src/flux/semantic/infer.rs

```diff
@@ -65,21 +65,21 @@ impl From<Constraint> for Constraints {
 pub fn solve(
     cons: &Constraints,
     with: &mut HashMap<Tvar, Vec<Kind>>,
-    f: &mut Fresher,
+    fresher: &mut Fresher,
```

Comment:
Thanks for making the names a little more semantic. 🙏

Reply:
I wish I could have reasonably done it for all of the single character variable names.